### ISSUE TITLE
Add a "Common" theme

### ISSUE THREAD

**Issue by web2style** [google/material-design-lite#861]
I'm sure the project will get much more popular if we add a "Common" theme. Material design is very unaccustomed for the majority and this stops lots of developers from using the project. If it doesn't have strict linking to Material Design, it could be very popular.

**Comment by addyosmani** [comment]
Could you provide some more details on what a "Common" theme might include that the current general text and article templates don't currently provide?

Any URLs to good examples of this type of page that you may have come across would also help us understand how we can better serve this need. Thanks!

**Comment by web2style** [comment]
My suggestion is not about templates (HTML), but about themes (CSS). I mean we can provide alternative CSS styles which look more familiar than Material Design. For example like Twitter Bootstrap or smth like this.

**Comment by surma** [comment]
What would you change about our [customizer](http://www.getmdl.io/customize/index.html) to fit your suggestion?

**Comment by Garbee** [comment]
I'm not sure a "common" theme fits the projects goals. Making things look more like "Bootstrap" is exactly what MDL isn't about. We adhere to the Material Design specification, if that isn't what you want then this project simply isn't for you in my opinion.

**Comment by web2style** [comment]
@surma, @Garbee There are lots of developers, who are sick of Bootstrap because it doesn't use any CSS Methodology (its huge classes specificity is pain to support, override and modify). But Bootstrap is so popular now, that it is used in almost every project, because it has lots of Open Source plugins, themes, implementations for popular JS frameworks, etc.

MDL is a gulp of fresh air after using Bootstrap (like gulp after grunt), because it uses BEM and Components. But Material Design itself is not so popular now, why don't we add a "Classic" theme? For now the MDL project is tightly linked to Material Design, why don't make it so universal and popular as Bootstrap?

For example: many developers won't consider MDL to make a quick prototype of their new project because of it's unusual design and lack of time for editing styles. If they don't use it, they won't contribute, write tutorials, etc.

This is what I mean.

**Comment by Garbee** [comment]
> For now the MDL project is tightly linked to Material Design, why don't make it so universal and popular as Bootstrap?

I don't think adding a "Bootstrap" theme will help with this (Not calling it classic anymore, since nothing is classic.) All it will do is take more resources to build and maintain from the core team.

If Developers chose MDL, they should be thinking about the MD aspect from the beginning. Not hacking something together that looks like Bootstrap but isn't. Yes, it looks different then what they are used to, that is the point. We shouldn't help by covering it up. Thinking differently is a good thing along with getting outside of your comfort zone.

**Comment by sgomes** [comment]
Thanks for the feedback, @web2style!

Sadly, MDL was designed from the ground up as a Material Design implementation, and most of the work we did on it revolved around implementing components as closely to the spec as possible. We're not in a good position to have different styling because that was never a goal for the project.

Along the way we did develop some components that are somewhat independent from any styling, though, the easiest example being the grid. These would actually be relatively straightforward to compile independently and use outside of any Material Design styling, and if we see a lot of demand in the community for those components, we will consider having separate downloads for them.
